dear-imgui.cabal

```diff
@@ -236,11 +237,11 @@ executable test
   import: common
   main-is: Main.hs
   default-language: Haskell2010
-  ghc-options: -Wall
+  ghc-options: -Wall 
```

Comment:
nit: extra trailing space

Reply:
Ah, error from switching from and to debug mode. Btw: is there an alternative to manually edit the cabal file to enable
```
ghc-options: -Wall -g -rtsopts -dcore-lint -debug -O0
cc-options: -g -O0
```
or would it be better to implement a debug-mode?